src/components/auto-field.tsx

```diff
@@ -44,16 +47,18 @@ export type AutoFormValueInfo = AutoFormCommonInfo &
   );
 
 export interface AutoFormCommonInfo {
-  label: string;
+  label?: string;
   description?: string;
   defaultValue?: any;
 }
 
+export interface AutoFormTextValueInfo {
+  type: AutoFormValueType.string;
+  inputProps?: InputProps;
+}
+
 export interface AutoFormSimpleValueInfo {
-  type:
-    | AutoFormValueType.string
-    | AutoFormValueType.number
-    | AutoFormValueType.uuid;
+  type: AutoFormValueType.number | AutoFormValueType.uuid;
```

Comment:
do we plan adding `inputProps?: InputProps;` to ` AutoFormValueType.number` too?